### DISCUSSION TITLE
Ability to self-host without deta base

### DISCUSSION THREAD

**Discussion by benoitjpnet** [WebCrateApp/webcrate#42] [Q&A] [answered]
Links to open feature request or ask community are broken so I use the classic bug report.

Hi,

Is there any plan to self-host the app without the need of deta base or any others third parties?

Thanks

**Comment by luxpal** [comment]
I agree

**Comment by BetaHuhn** [comment, marked as answer]
Hey, sorry for the broken feedback link. Should be fixed now.

There are currently no plans to move away from Deta and Deta services like Base. It powers the whole backend, including authentication and the database, and would require a complete rewrite of the app.

**Reply by MikePadge** [reply]
Oh lol, glad I stumbled upon this, I'm over here trying to just build and run the node application... project key not defined when trying to hit base... because it's "deta" 

Super awesome concept, slick design, looks fantastic... I am not going anywhere near deta. I don't care how super secure it is, and that you can't see my data in it, I don't care. I believe you to the extent that I trust any code, and I trust it more if I'm running it somewhere I can break it myself. 

This is going to destroy any type of larger community collaboration you may have been hoping to get for this project.

If you even care about that sort of thing. 

....welp, I guess back to something less cool looking. Good luck with this.

**Reply by jjonesdesign** [reply]
I was looking to use this software, but I'm in the same boat. If you don't control the data, you down own it.

The project looks perfect, but I guess I'll have to find another solution, unfortunately.

**Reply by binaryben** [reply]
Bugger. Please make that boat a bit bigger so I can squeeze in @jjonesdesign? 😅 Awesome looking UI - props to the design work! Will not touch it though if I can't self host it using Docker or similar. The fact the auth requires Deta is even worse for me as I'd be hoping to integrate it with Authelia/LDAP for SSO.

I'm currently eyeing [Omnivore](https://omnivore.app/) for the read it later functionality. Open to other recommendations if anyone has any though...

**Comment by ndom91** [comment]
Yeah +1 to this.. Would love to self-host this on my own! Until then, I'm going to continue to look..

**Comment by darkworks** [comment]
seems like we need fork of this project and add ability of self hosting

**Reply by ndom91** [reply]
Maybe check out https://github.com/ndom91/briefkasten as a simpler to self-host alternative. 

Full transparency, I'm the author 😅
